File: Modules/University/University.Application/DTOs/FacultyDto.cs
namespace University.Application.DTOs
{
    public class FacultyDto
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string? Address { get; set; }
        public string Code { get; set; } = string.Empty;
    }
}

File: Frontend/src/dto/FacultyDTO.ts
export type FacultyResponseDTO = {
  id: string; // GUID from backend
  name: string;
  address: string;
  code: string;
  createdAt: string;
  updatedAt?: string;
};

export type CreateFacultyRequestDTO = {
  name: string;
  address: string;
  code: string;
};

export type UpdateFacultyRequestDTO = CreateFacultyRequestDTO;
